README.md

```diff
@@ -51,7 +60,13 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+var pizza = {
+  temperature: 70
+  function bake(bakeInput) {
+    temperature = bakeInput
```

Comment:
This would create a global variable called `temperature`. You want instead to change `pizza.temperature`. You could do that by saying `pizza.temperature = bakeInput`, or `this.temperature = bakeInput` since `bake` is a method of the `pizza` object.